src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -25,7 +25,8 @@
 <body>
 
   <nav>
-    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a id="navTitle" href="/">
+    <img src = "https://drive.google.com/uc?id=1dG9V-sBNMS9hEivT4L-sn1M0m7RIn0Gr" width="160" height="80" /></a>
```

Comment:
You could consider putting these image files directly inside the project. Not a big deal though as long as this works!